src/openai_cli/config.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+DEFAULT_MODEL = "gpt-4o-mini"
+MAX_TOKENS = 500
+TEMPERATURE = 0.23
```

Comment:
I know it wasn't possible before, but since you're rewriting almost all of it, could you please add a CLI option to set temperature and max tokens. Again, this is optional, up to you.